backend/routes/PostReaction/AttendeeHelperFunction.py

```
@@ -0,0 +1,59 @@
+def get_event_by_id(db: Session, event_id: int) -> Event:
+    """Fetch an event by ID or raise 404."""
+    event = db.query(Event).filter(Event.id == event_id).first()
+    if not event:
+        raise HTTPException(status_code=404, detail="Event not found.")
+    return event
+
+def get_user_rsvp(db: Session, event_id: int, user_id: int) -> EventAttendee:
+    """Fetch the RSVP status of a user for an event."""
+    return db.query(EventAttendee).filter(
+        EventAttendee.event_id == event_id,
+        EventAttendee.user_id == user_id
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)